### ISSUE TITLE
Design Changes in /paint/ page 

### ISSUE THREAD

**Issue by 7saikat7** [FOSS-Cell-GECPKD/pixelvibe#405]
In the paint page, we need some kind of better design in terms of the position of the pencil size, delete buttons, so you can come up with a better design format for that page that will be more user-friendly and also handy to use.

**Do not forget to attach a screenshot of your new design with the PR **

**Comment by Amanmawar17** [comment]
i'm interested

**Comment by ankitapuri** [comment]
@Amanmawar17 I'll assign it to u . Deadline - 11/12/2021

**Comment by ankitapuri** [comment]
@Amanmawar17 any updates?

**Comment by Amanmawar17** [comment]
@ankitapuri i am working.....

**Comment by Amanmawar17** [comment]
This issues is taking more time as specified deadline. So I request to extend deadline.

**Comment by 7saikat7** [comment]
> This issues is taking more time as specified deadline. So I request to extend deadline.

Ok, @Amanmawar17, you can take 2 more days, but please try to finish it within that only.

**Comment by 7saikat7** [comment]
Create PR only in dev branch

**Comment by AryanShinde** [comment]
Hey, If this issue is not solved yet, I am interested in it. I'll share a Figma design before coding it directly.

**Comment by ankitapuri** [comment]
@AryanShinde currently @Amanmawar17 is working on it . if he's unable to do u can take it. 

@Amanmawar17 try doing it by this Sunday .

**Comment by Amanmawar17** [comment]
@ankitapuri it's password is still not working with normal i'ds please resolve this issue

**Comment by ankitapuri** [comment]
> @ankitapuri it's password is still not working with normal i'ds please resolve this issue


@arpit456jain can u please look into it.

**Comment by arpit456jain** [comment]
ok

**Comment by arpit456jain** [comment]
@ankitapuri @Amanmawar17 i had checked the login functionality its working fine from my side 

https://user-images.githubusercontent.com/55352601/146591574-15764a12-b071-48d6-92e7-6cdde2d5aca6.mp4

**Comment by arpit456jain** [comment]
@Amanmawar17 please share a video where u r getting problem

**Comment by ankitapuri** [comment]
@Amanmawar17 its working fine for me too

**Comment by Amanmawar17** [comment]
@ankitapuri @arpit456jain  Yeah its working i send pull request till 7.30 in evening (Today)
